crates/sui-indexer-alt-framework/src/pipeline/concurrent/mod.rs

```diff
@@ -285,6 +319,61 @@ const fn max_chunk_rows<H: Handler>() -> usize {
     }
 }
 
+/// Starts a task for a tasked pipeline to track the main reader lo.
+pub(super) fn main_reader_lo_task<H: Handler + 'static>(
```

Comment:
Let's move this into its own module -- that's what we have done for all the other tasks.